spec/routes/directory.spec.js

```diff
@@ -49,7 +49,7 @@ describe('directory route', function () {
         });
     });
 
-    it('returns multiple employees', function (done) {
+    it.only('returns multiple employees', function (done) {
```

Comment:
Why are you using [`only`](https://mochajs.org/#exclusive-tests)? I'm pretty sure we don't want to skip the other tests in the suite. 😉